tests/themis/themis_secure_session.c

```diff
@@ -50,6 +50,108 @@ static uint8_t server_pub[] = {
     0x6e, 0xd6, 0xac, 0x88, 0xb6, 0x44, 0xc2, 0x9a, 0x24, 0x84, 0xee, 0x50, 0x4c, 0x3e, 0xa0,
 };
 
+static uint8_t client_priv_rsa[] = {
+    0x52, 0x52, 0x41, 0x32, 0x00, 0x00, 0x04, 0x90, 0x94, 0x65, 0xbe, 0x5c, 0x75, 0x5e, 0x6a,
+    0xea, 0xeb, 0x7d, 0x0d, 0x59, 0xa4, 0xe3, 0xc4, 0xb0, 0x2c, 0x11, 0x7b, 0xc6, 0xb0, 0x0e,
```

Comment:
While this is still in draft, is it possible to make these keys more compact? For example, using hex strings, like here:
https://github.com/cossacklabs/themis/blob/86d9e705fc55c20b39d23d370110954c93a33d34/tests/themis/themis_secure_cell.c#L34-L35
Horizontal scroll and over 9000 column lines may not be ideal, but it definitely beats scrolling vertically through a data arrays that is never going to be edited.

Reply:
Even when using `"\x43\x68\x75\x75\x30..."`-like syntax with 24 bytes per line, it takes 49 lines and 102 columns to store RSA private key. Maybe I'll move test keys to a different file instead. What about `themis_secure_session_test_keys.h`?

Reply:
Seems an okay name, go for it 👍